airavata-api/src/main/java/org/apache/airavata/helix/impl/task/aws/AWSCompletingTask.java

```diff
@@ -1,21 +1,22 @@
 /**
- * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
The only changes to this file are from mvn spotless:apply.

Otherwise, the build won't pass